confluence-slack-server-integration-plugin/src/main/java/com/atlassian/confluence/plugins/slack/spacetochannel/listener/SlackChannelEventListener.java

```diff
@@ -66,10 +70,14 @@ public void onConversationsLoadedEvent(final SlackConversationsLoadedEvent event
         // unmute channel if they are returned in the list of active channels;
         // just in case we missed 'channel_unarchive' event
         asyncExecutor.run(() -> {
-            final List<String> mutedChannelIds = settingService.getMutedChannelIds();
+            final List<ConversationKey> mutedChannelIds = settingService.getMutedChannelIds();
             if (!mutedChannelIds.isEmpty()) {
-                Set<String> activeConversationIds = event.getConversations().stream()
-                        .map(Conversation::getId)
+                Set<ConversationKey> activeConversationIds = event.getConversations().stream()
+                        .flatMap(c -> {
+                            List<ConversationKey> list = new ArrayList<>();
```

Comment:
Same comment as the other listener above, i.e. please add getTeamId to `SlackConversationsLoadedEvent`.